enable-vcp-image/enable-vcp-scripts/manager_pod.sh

```diff
@@ -7,14 +7,36 @@ source "$(dirname "$0")"/exit_codes.sh
 # read secret keys from volume /secret-volume/ and set values in an environment
 read_secret_keys
 
+create_daemonset () {
+        version=`kubectl version --short --output json`
+        minor=`echo $version | jq '.serverVersion.minor' | sed 's/"//g'`
+        major=`echo $version | jq '.serverVersion.major' | sed 's/"//g'`
+
+        resource_yaml=""
+        if [ $minor -ge 9 ] && [ "$major" -ge 1 ]; then
```

Comment:
Customer recently reported that v1.8.6 also does not support `v1beta1.ThirdPartyResource`. Customer observation is 
```
error: error validating "/opt/enable-vcp-scripts/vcp-daemontset.yaml": error validating data: couldn't find type: v1beta1.ThirdPartyResource; if you choose to ignore these errors, turn validation off with --validate=false
```
Issue: https://github.com/vmware/kubernetes/issues/432